appscale/tools/appscale_stats.py

```diff
@@ -0,0 +1,655 @@
+    )
+    print_table(
+      table_name="SUMMARY PROCESS STATISTICS"
+      if not options.verbose else "PROCESS STATISTICS",
+      headers=process_headers,
+      data=process_stats
+    )
+    if process_failures:
+      failures["processes"] = process_failures
+
+  if "proxies" in options.show:
+    raw_proxy_stats, proxy_failures = _get_stats(
+      keyname=options.keyname,
+      stats_kind="proxies",
+      include_lists=INCLUDE_PROXY_LIST
+    )
+    proxy_headers, proxy_stats = get_proxy_stats(
+      raw_proxy_stats=raw_proxy_stats,
+      verbose=options.verbose,
+      apps_filter=options.apps_only
+    )
+    proxy_stats = sort_proxy_stats(
+      proxy_stats=proxy_stats,
+      column=0
+    )
+    print_table(
+      table_name="PROXY STATISTICS",
+      headers=proxy_headers,
+      data=proxy_stats
+    )
+    if proxy_failures:
+      failures["proxies"] = proxy_failures
+
+  if failures:
+    print_failures(failures=failures)
+
+
+def get_marked(data, mark):
+  """
+  Marks data in a specified mark.
+
+  Args:
+    data: An object to be marked in.
+    mark: A string representing one of the existing marks
+      ('red', 'green' or 'bold').
+
+  Returns:
+    A string marked in.
+  """
+  marks = {
+    "red": "\033[91m",
+    "green": "\033[92m",
+    "bold": "\033[1m"
+  }
+  return marks[mark] + str(data) + "\033[0m"
+
+
+def render_loadavg(loadavg):
+  """
+  Renders loadavg information.
+
+  Args:
+    loadavg: A dict representing loadavg values.
+
+  Returns:
+    A string with information about node loadavg
+    last 1, 5 and 15 minuts in format: "last 1min / last 5min / last 15min"
+    and marked in red if loadavg is more than 2.0.
+  """
+  limit_value = 2.0
+  last_1 = loadavg["last_1min"]
+  last_5 = loadavg["last_5min"]
+  last_15 = loadavg["last_15min"]
+
+  return "{} / {} / {}".format(
+    last_1 if last_1 < limit_value else get_marked(last_1, "red"),
+    last_5 if last_5 < limit_value else get_marked(last_5, "red"),
+    last_15 if last_15 < limit_value else get_marked(last_15, "red")
+  )
+
+
+def render_partitions(partitions, verbose):
+  """
+  Renders partitions information.
+
+  Args:
+    partitions: A dict representing partition values.
+    verbose: A boolean - render all partitions if True,
+      add only three the most used partitions if False.
+
+  Returns:
+    A string with information about partition values
+      marked partition and value in red if value > 90%.
+  """
+  part_list = [[part, 100 * value["used"] / value["total"]]
+               for part, value in
+               partitions.iteritems()]  # calculate percents
+
+  # sort from more used to less used
+  part_list.sort(key=lambda p: p[1], reverse=True)
+
+  partitions_info = [
+    "'{}': {}%".format(part[0], part[1])
+    if part[1] < 90
+    else get_marked("'{}': {}%".format(part[0], part[1]), "red")
+    for part in part_list
+  ]
+
+  if not verbose:
+    if len(partitions_info) > 3:
```

Comment:
Alternatively you could write `if not verbose and len(partitions_info) > 3:`. JFYI (Just for your information)